pkg/apis/sessions/session_state_test.go

```diff
@@ -17,10 +17,14 @@ func timePtr(t time.Time) *time.Time {
 	return &t
 }
 
+func NewCipher(secret []byte) (encryption.Cipher, error) {
```

Comment:
This should be private, and given it's for testing purposes, how about `newTestCipher` so that we know it's meant only for testing?